docs/pages/components/modal.mdx

```diff
@@ -93,21 +96,28 @@ function() {
 
 ## Header and Footer
 
-Use `Modal.Title` and `Modal.Footer` to style your Modal.
+Use `Modal.Header` and `Modal.Footer` to style your Modal.
 
 ```jsx
 function() {
   const modal = useModalState()
+  const title = "Nullam non lacinia"
+  const subtitle = "Praesent sit amet quam ac velit faucibus dapibus, quisque sapien ligula."
+  const icon = <ActionsIcon size="xl" color="dark.900" />
+
+  const informations = {
+    title: "Donec id tellus",
+    subtitle: "Sed lorem lacus, hendrerit non sodales id, consectetur quis magna. Mauris auctor blandit porta. In imperdiet rutrum nunc."
+  }
+
```

Comment:
Can you please add more examples with differents header and footer ? or create a selector to show/not show informations ?